daemon/algod/api/algod.oas2.json

```
@@ -3518,6 +3518,10 @@
         "stack-change": {
           "description": "A boolean option enabling returning stack changes together with execution trace during simulation.",
           "type": "boolean"
+        },
+        "scratch-change": {
+          "description": "A boolean option enabling returning scratch slot changes.",
```

I suppose we should use the same language for both options:
```suggestion
          "description": "A boolean option enabling returning scratch slot changes together with execution trace during simulation.",
```